src/Models/Test/ModelInstantiatorTestDBModel.php

```diff
@@ -21,11 +21,17 @@ public function getAttributes(): array
     }
 
     // Mimic a ManyToOne relation (object).
-    public function addOneRelation(ModelInstantiatorTestDBModel $relation): void
+    public function addOneRelation(?ModelInstantiatorTestDBModel $relation): void
```

Comment:
Had to add the `?` here because I wanted to be able to test that the exception is thrown by the `ModelInstantiator` so this function needed to be able to accept `null`.